kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/PartitionDeserializer.java

```diff
@@ -0,0 +1,26 @@
+package org.axonframework.extensions.kafka.eventhandling;
+
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.KeyDeserializer;
+import org.apache.kafka.common.TopicPartition;
+
+/**
+ * JacksonSerializer handle Map that key belong TopicPartition must handle with special deserializer
+ * For more information see:
+ *     https://github.com/AxonFramework/extension-kafka/issues/34
```

Comment:
Although correct, I'd recommend a different wording for this deserializer:

```suggestion
 * Custom {@link KeyDeserializer} used to deserialize the {@link TopicPartition}.
 * 
```